addons/stock/models/stock_quant.py

```diff
@@ -327,6 +330,8 @@ def action_view_inventory(self):
         return action
 
     def action_apply_inventory(self):
+        # Compute `inventory_diff_quantity` if quantity was not set manually
+        self.filtered(lambda quant: not quant.inventory_quantity_set).inventory_quantity_set = True
```

Comment:
I did that because calling `action_apply_inventory` with `inventory_quantity_set` set to False is valid and will set the quantity to 0.
However, with my changes, it doesn't work anymore since the difference is always 0 when `inventory_quantity_set` is not set.

It was to keep the behaviour for customisation…
Should I remove this hack?